source/ada/lsp-ada_handlers.adb

```diff
@@ -3629,15 +3630,32 @@ package body LSP.Ada_Handlers is
          end if;
       end On_Inaccessible_Name;
 
-      Query : constant VSS.Strings.Virtual_String :=
-        Canonicalize (LSP.Types.To_Virtual_String (Request.params.query));
+      Pattern : Search_Pattern_Access := Build
+        (Pattern        => LSP.Types.To_Virtual_String (Request.params.query),
+         Case_Sensitive =>
+           (if Request.params.case_sensitive.Is_Set
```

Comment:
We can compare `Request.params.case_sensitive` to `LSP.Types.True` and it will be a bit shorter:

    Case_Sensitive => Request.params.case_sensitive = LSP.Types.True,

Reply:
done